py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -152,16 +170,22 @@ def layer_norm_no_plugin(
     name: str,
     input: TRTTensor,
     normalized_shape: List[int],
-    weight: torch.Tensor,
-    bias: torch.Tensor,
-    eps: List[float],
+    weight: Optional[Union[TRTTensor, torch.Tensor, np.ndarray]],
+    bias: Optional[Union[TRTTensor, torch.Tensor, np.ndarray]],
```

Comment:
`TRTTensor` would not be a valid input here, for the scale layer

Reply:
Do you mean the type of `weight` and `bias` in all the three functions should be `Optional[Union[torch.Tensor, np.ndarray]]`?  I see its native function: 
```
func: layer_norm(Tensor input, SymInt[] normalized_shape, Tensor? weight=None, Tensor? bias=None, float eps=1e-05, bool cudnn_enable=True) -> Tensor
```

Reply:
Yes, I think it should be `ptional[Union[torch.Tensor, np.ndarray]]`, because if either of those is a `TRTTensor`, the computation below would not work (`to_numpy` can't be called on a `TRTTensor`)